docs/Routes.md

```diff
@@ -270,3 +271,31 @@ fastify.get('/it', { config: { output: 'ciao mondo!' } }, handler)
 
 fastify.listen(3000)
 ```
+
+<a name="version"></a>
+### Version
+If needed you can provide a version option, which will allow you to declare multiple versions of the same route. The versioning should follow the [semver](http://semver.org/) specification.<br/>
+Fastify will automatically detect the `Accept-Version` header and route the request accordingly (advanced ranges and pre-releases currently are not supported).<br/>
+*Be aware that using this feature will cause a degradation of the overall performances of the router.*
+```js
+fastify.route({
+  method: 'GET',
+  url: '/',
+  version: '1.2.0',
+  handler: function (request, reply) {
+    reply.send({ hello: 'world' })
+  }
+})
+
+fastify.inject({
+  method: 'GET',
+  url: '/',
+  headers: {
+    'Accept-Version': '1.x' // it could also be '1.2.0' or '1.2.x'
```

Comment:
`// it could also be '1.2.0' or '1.2.x'`=> `// it could also be '1.2.0' or '1.2.x' but not '1.1x'`?

Reply:
As far as I know, `1.1x` is not a valid semver string.

Reply:
Sorry '1.1.x'

Reply:
**Any** basic semver range is valid, in the comment are just examples.